### PR TITLE
Update nightly to 1.26.0 and bootstrap from beta.

### PR DESCRIPTION
**Pull request by Mark-Simulacrum**

### CODE REVIEW

File: src/stage0.txt [hunk 1]
```diff
@@ -12,7 +12,7 @@
 # source tarball for a stable release you'll likely see `1.x.0` for rustc and
 # `0.x.0` for Cargo where they were released on `date`.
 
-date: 2018-01-02
+date: 2018-01-18
```
Comment (Keruspe) [inline src/stage0.txt]: Shouldn't that be 2018-02-18 rather that 01?

Comment (Mark-Simulacrum) [reply src/stage0.txt]: Yes, thank you.

**Comment by kennytm** [comment]
@bors retry 

`dist-{armv7,armhf}-linux` 3-hour timed out.

**Comment by kennytm** [comment]
@bors retry 

3 hour timeout

**Pull request merged**